10245.xml

```diff
@@ -1,14 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<!-- MIT License
-Copyright (c) 2017 IPSO Alliance
-Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, 
-including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
-The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. 
-IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, 
-DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.-->
+<!-- BSD-3 Clause License
 
+Copyright 2019 Open Mobile Alliance. 
```

Comment:
The name of the company cannot be Open Mobile Alliance, it should be Vodafone.
This comment applies to every object changed by this PR

Reply:
This original submission was submitted under the MIT Copyright (c) 2017 IPSO Alliance, so shouldn't this transfer to OMA?

Reply:
In that case, there was an original mistake on the name of the company. If we got an agreement from Vodafone that they are ok to change the license from MIT to BSD-3, we should also ensure that they are ok to have their name on the Objects.

Reply:
Vodafone company details added